internal/core/software_renderer.rs

```diff
@@ -597,13 +597,15 @@ struct SceneTexture<'a> {
     stride: u16,
     source_size: PhysicalSize,
     color: Color,
+    alpha: f32,
```

Comment:
not f32 in the software renderer.  Use u8.

Reply:
ok will change that on this position

Reply:
To elaborate, this part of the code is very sensitive in both memory and performence.  These mutlitplication will happen for every pixel, on hardware that might not have a floating point unit.  So f32 operations for each pixel is really a no-go.